src/main/java/edu/cornell/kfs/coa/batch/CreateClosedLaborAccountsCsvByPreviousDaysStep.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.kfs.coa.batch;
+
+import java.util.Date;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.cornell.kfs.coa.batch.service.CreateClosedLaborAccountsCsvByPreviousDaysService;
+
+public class CreateClosedLaborAccountsCsvByPreviousDaysStep extends AbstractStep {
```

Comment:
I think this step class name is too wordy, How about just naming the class "CreateClosedLaborAccountsCsvStep"